components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1352,6 +1353,9 @@ struct TransactionInfo {
   OriginInfo? origin_info;
   string? group_id;
   string chain_id;
+  // Currently only available for ETH and FIL
+  // TODO(cypt4): Add for SOL
+  string? final_recipient;
```

Comment:
Is 'final' a correct word for that field? Should that be 'effective'? Or?
@onyb

Reply:
Or just `recipient` without any extra qualifiers. Also, this field should be commented for the various supported cases (fil forward, erc20 transfer, etc.), and the difference from `to` field should be mentioned. @cypt4